sros2/setup.py

```diff
@@ -12,11 +12,11 @@ def package_files(directory):
     return paths
 
 
-extra_files = []
-extra_files.extend(package_files('sros2/policy/defaults'))
-extra_files.extend(package_files('sros2/policy/schemas'))
-extra_files.extend(package_files('sros2/policy/templates'))
-
+extra_files = (
```

Comment:
Is this change relevant, and if so how?

Reply:
It allows mypy to determine the type without us needing to add it manually.